whylog/config/super_parser.py

```diff
@@ -0,0 +1,62 @@
+import re
+from abc import ABCMeta, abstractmethod
+
+import six
+
+from whylog.converters import CONVERTION_MAPPING, STRING
+
+
+@six.add_metaclass(ABCMeta)
+class AbstractSuperParser(object):
+    @abstractmethod
+    def get_ordered_groups(self, line):
+        pass
+
+
+class RegexSuperParser(AbstractSuperParser):
+    NO_MATCH = []
```

Comment:
`tuple()` would he better - nobody will append nothing to it, can use less memory etc